### PR TITLE
cf-update-carter

### PR DESCRIPTION
**Pull request by kakax114**

### CODE REVIEW

File: content/community-fund/index.en.md [hunk 1]
```diff
@@ -7,15 +7,16 @@ description: "The NavCoin community fund is a pool of NAV that's set aside to he
 ---
 <section class="cf-hero">
     <div class="grid-container">
-      <div class="text-container">
-        <h1>Community Fund</h1>
-        <p>NavCoin is built by the community, for the community - it’s an open-source project that ran no premine or ICO. NavCoin’s Community Fund enables anyone to get paid by the network for their contributions to the NavCoin ecosystem. Projects are funded and approved by the decentralized network with no central authority. This ensures NavCoin’s direction remains firmly in the best interest of the network and is autonomously determined by the community.</p>
-      </div>
-      <img src="/images/community-fund/cfu-hero.png" alt="Hero image">
+        <div class="text-container">
+            <h1 style="font-weight: 600; margin-bottom:35px;">Community Fund</h1>
```
Comment (matt-auckland) [inline content/community-fund/index.en.md]:
```scss
.cf-hero .grid-container .text-container h1 { 
  font-weight: 600; 
  margin-bottom:35px;
}
```
Can you move this into the community fund scss?

File: content/community-fund/index.en.md [hunk 2]
```diff
@@ -73,16 +76,155 @@ description: "The NavCoin community fund is a pool of NAV that's set aside to he
     >}}
 {{< /item_container_section >}}
 
+<section class="item-container-section  bg-purple " id="">
```
Comment (matt-auckland) [inline content/community-fund/index.en.md]: I think you can probably throw this into an `item_container_section`
```html
{{<item_container_section 
  titleText="How The Fund Works In 4&nbsp;Steps">}}
    <!-- your four steps html -->
{{</item_container_section>}}
```

Comment (matt-auckland) [reply content/community-fund/index.en.md]: If the default template doesn't work than never mind

**Comment by kakax114** [comment]
win!

**Pull request merged**